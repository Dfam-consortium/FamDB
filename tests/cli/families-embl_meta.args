families
--format
embl_meta
-d
3
