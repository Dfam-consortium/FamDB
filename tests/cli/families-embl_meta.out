CC   <copyright header>
ID   DR0000001; SV 1; linear; DNA; STD; UNC; 8 BP.
NM   Test family DR0000001
XX
AC   DR0000001;
XX
XX
KW   /.
XX
OS   Drosophila <fungus>
OC   Third Clade.
XX
CC
CC   RepeatMasker Annotations:
CC        Type: 
CC        SubType: 
CC        Species: Drosophila_fungus
CC        SearchStages: 
CC        BufferStages: 
XX
//
ID   DR_Repeat1; SV 1; linear; DNA; STD; UNC; 7 BP.
NM   Test family DR_Repeat1
XX
AC   DR_Repeat1;
XX
XX
KW   /.
XX
OS   Drosophila <flies>
OC   Third Clade.
XX
CC
CC   RepeatMasker Annotations:
CC        Type: 
CC        SubType: 
CC        Species: Drosophila_flies
CC        SearchStages: 
CC        BufferStages: 
XX
//
ID   TEST0003; SV 1; linear; DNA; STD; UNC; 4 BP.
NM   Test family TEST0003
XX
AC   TEST0003;
XX
XX
KW   /.
XX
OS   Third Clade
OC   .
XX
CC
CC   RepeatMasker Annotations:
CC        Type: 
CC        SubType: 
CC        Species: Third_Clade
CC        SearchStages: 30,40
CC        BufferStages: 
XX
//
